docs/roadmap/index.md

```diff
@@ -1,15 +1,47 @@
 # Roadmap
 
-In our planning discussions we identified broad three areas of work:
+!!!hint "Production Ready"
+    The Flux custom resource definitions which are at `v1beta1` and `v2beta1`
+    and their controllers are considered stable and production ready.
+    Going forward, changes to the stable CRDs will be accompanied by a conversion mechanism.
```

Comment:
```suggestion
    Going forward, breaking changes to the stable CRDs will be accompanied by a conversion mechanism.
```

maybe? As we make changes that are non-breaking without offering a conversion mechanism.

Reply:
good point! thanks